index.bs

```diff
@@ -19,8 +19,10 @@ Markup Shorthands: markdown on
 Inline Github Issues: true
 !Test Suite: <a href="https://github.com/web-platform-tests/wpt/tree/main/magnetometer">web-platform-tests on GitHub</a>
 Boilerplate: omit issues-index, omit conformance, repository-issue-tracking no
-Include MDN Panels: if possible
```

Comment:
Was removing the MDN panels intentional?

Reply:
Yes, see https://github.com/w3c/magnetometer/pull/62#issuecomment-985448111